frontend/src/v5/ui/routes/dashboard/projects/tickets/ticketsTable/ticketsTableContent/ticketsTableContent.component.tsx

```diff
@@ -30,26 +30,57 @@ import { useEdgeScrolling } from '../edgeScrolling';
 import { BaseProperties } from '@/v5/ui/routes/viewer/tickets/tickets.constants';
 import { INITIAL_COLUMNS } from '../ticketsTable.helper';
 import { useContextWithCondition } from '@/v5/helpers/contextWithCondition/contextWithCondition.hooks';
+import { Transformers, useSearchParam } from '@/v5/ui/routes/useSearchParam';
+import { isEqual } from 'lodash';
 import { TicketsFiltersContext } from '@components/viewer/cards/cardFilters/ticketsFilters.context';
 
 const TableContent = ({ template, tableRef, ...props }: TicketsTableResizableContentProps & { template: ITemplate, tableRef }) => {
 	const edgeScrolling = useEdgeScrolling();
 	const {
 		stretchTable, getAllColumnsNames, subscribe, resetWidths,
 		setVisibleSortedColumnsNames,
+		getVisibleSortedColumnsNames,
 	} = useContextWithCondition(ResizableTableContext, []);
-	const { isFiltering } = useContext(TicketsFiltersContext)
+	const { isFiltering } = useContext(TicketsFiltersContext);
 	const templateWasFetched = templateAlreadyFetched(template);
+	const ignoreColumnChange = useRef(false);
+	const [colsParam, setColsParams] = useSearchParam('cols', Transformers.STRING_ARRAY, true);
+
+	const setVisibleColumn = () => {
+		ignoreColumnChange.current = true;
+		
+		if (!colsParam.length) {
+			const allColumns = getAllColumnsNames();
+			const initialColumns = INITIAL_COLUMNS.filter((name) => allColumns.includes(name));
+
+			if (!isEqual(getVisibleSortedColumnsNames(), initialColumns)) {
+				setVisibleSortedColumnsNames(initialColumns);
+				resetWidths();
+				stretchTable(BaseProperties.TITLE);
```

Comment:
A minor point perhaps for a future issue, but if the title column is excluded when sharing the link the table will not stretch to fit the screen. Perhaps it should check to see if the title is included in the table and if not fall back on stretching the first or last column